src/grants/auth_code.grant.ts

```diff
@@ -118,10 +118,8 @@ export class AuthCodeGrant extends AbstractAuthorizedGrant {
     return await this.makeBearerTokenResponse(client, accessToken, scopes, extraJwtFields);
   }
 
-  canRespondToAuthorizationRequest(request: RequestInterface): boolean {
-    const responseType = this.getQueryStringParameter("response_type", request);
-    const hasClientId = !!this.getQueryStringParameter("client_id", request);
-    return responseType === "code" && hasClientId;
+  handlesResponseTypeForAuthorizationRequest(request: RequestInterface): boolean {
```

Comment:
I've renamed the function to reflect what it necessarily actually does now

Reply:
The only thing I changed in your PR was I just reverted this rename. I appreciate the attempt, I just think the previous name still makes more sense in my head right now.